testing/unit/test_sdf.py

```diff
@@ -99,6 +110,15 @@ def test_aromatic(self):
                 aromatic_bond = True
         assert(aromatic_bond)
 
+    def test_charges(self):
```

Comment:
Can you add a docstring for this? With most of the work being done in read_write_read, its not obvious what this is testing